buildbot.tac

```diff
@@ -23,7 +26,3 @@ from flocker_bb.upgrade import UpgradeService
 
 master = BuildMaster(basedir, configfile)
 UpgradeService(basedir, configfile, master).setServiceParent(application)
-
-
-from twisted.application.internet import TimerService
-TimerService(30, master.botmaster.maybeStartBuildsForAllBuilders).setServiceParent(master)
```

Comment:
Yes. It doesn't matter if we try to start a build every 30s, if buildbot doesn't think there is anywhere it can run things.